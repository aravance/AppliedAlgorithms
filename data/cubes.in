1 20
21 23
24 36
1 5
6 6
7 17
18 18
12 179
180 180
180 200
0 0
