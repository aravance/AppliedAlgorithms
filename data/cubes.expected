6 = f(3,4,5)
9 = f(1,6,8)
12 = f(6,8,10)
18 = f(2,12,16)
18 = f(9,12,15)
19 = f(3,10,18)
20 = f(7,14,17)
+++
+++
24 = f(12,16,20)
25 = f(4,17,22)
27 = f(3,18,24)
28 = f(18,19,21)
29 = f(11,15,27)
30 = f(15,20,25)
36 = f(4,24,32)
36 = f(18,24,30)
+++
+++
6 = f(3,4,5)
+++
9 = f(1,6,8)
12 = f(6,8,10)
+++
18 = f(2,12,16)
18 = f(9,12,15)
+++
12 = f(6,8,10)
18 = f(2,12,16)
18 = f(9,12,15)
19 = f(3,10,18)
20 = f(7,14,17)
24 = f(12,16,20)
25 = f(4,17,22)
27 = f(3,18,24)
28 = f(18,19,21)
29 = f(11,15,27)
30 = f(15,20,25)
36 = f(4,24,32)
36 = f(18,24,30)
38 = f(6,20,36)
40 = f(14,28,34)
41 = f(2,17,40)
41 = f(6,32,33)
42 = f(21,28,35)
44 = f(16,23,41)
45 = f(5,30,40)
46 = f(3,36,37)
46 = f(27,30,37)
48 = f(24,32,40)
50 = f(8,34,44)
53 = f(29,34,44)
54 = f(6,36,48)
54 = f(12,19,53)
54 = f(27,36,45)
56 = f(36,38,42)
57 = f(9,30,54)
58 = f(15,42,49)
58 = f(22,30,54)
60 = f(21,42,51)
60 = f(30,40,50)
63 = f(7,42,56)
66 = f(33,44,55)
67 = f(22,51,54)
69 = f(36,38,61)
70 = f(7,54,57)
71 = f(14,23,70)
72 = f(8,48,64)
72 = f(34,39,65)
72 = f(36,48,60)
75 = f(12,51,66)
75 = f(38,43,66)
76 = f(12,40,72)
76 = f(31,33,72)
78 = f(39,52,65)
80 = f(28,56,68)
81 = f(9,54,72)
81 = f(25,48,74)
82 = f(4,34,80)
82 = f(12,64,66)
82 = f(19,60,69)
84 = f(28,53,75)
84 = f(42,56,70)
84 = f(54,57,63)
85 = f(50,61,64)
87 = f(20,54,79)
87 = f(26,55,78)
87 = f(33,45,81)
87 = f(38,48,79)
88 = f(21,43,84)
88 = f(25,31,86)
88 = f(32,46,82)
89 = f(17,40,86)
90 = f(10,60,80)
90 = f(25,38,87)
90 = f(45,60,75)
90 = f(58,59,69)
92 = f(6,72,74)
92 = f(54,60,74)
93 = f(32,54,85)
95 = f(15,50,90)
96 = f(19,53,90)
96 = f(48,64,80)
97 = f(45,69,79)
99 = f(11,66,88)
100 = f(16,68,88)
100 = f(35,70,85)
102 = f(51,68,85)
103 = f(12,31,102)
105 = f(33,70,92)
106 = f(58,68,88)
108 = f(12,72,96)
108 = f(13,51,104)
108 = f(15,82,89)
108 = f(24,38,106)
108 = f(54,72,90)
110 = f(29,75,96)
111 = f(16,47,108)
112 = f(72,76,84)
113 = f(50,74,97)
114 = f(18,60,108)
114 = f(57,76,95)
115 = f(3,34,114)
116 = f(23,86,97)
116 = f(30,84,98)
116 = f(44,60,108)
117 = f(13,78,104)
120 = f(9,55,116)
120 = f(42,84,102)
120 = f(60,80,100)
121 = f(49,84,102)
122 = f(19,92,101)
123 = f(6,51,120)
123 = f(18,96,99)
123 = f(44,51,118)
125 = f(20,85,110)
126 = f(14,84,112)
126 = f(23,94,105)
126 = f(63,84,105)
127 = f(13,65,121)
129 = f(38,57,124)
132 = f(5,76,123)
132 = f(48,69,123)
132 = f(66,88,110)
133 = f(21,70,126)
134 = f(44,102,108)
134 = f(86,95,97)
135 = f(15,90,120)
137 = f(44,73,128)
138 = f(9,108,111)
138 = f(69,92,115)
138 = f(72,76,122)
138 = f(81,90,111)
139 = f(94,96,99)
140 = f(14,108,114)
140 = f(49,98,119)
140 = f(90,95,105)
141 = f(72,85,122)
142 = f(28,46,140)
142 = f(31,64,137)
144 = f(1,71,138)
144 = f(16,96,128)
144 = f(68,78,130)
144 = f(72,96,120)
145 = f(12,81,136)
145 = f(55,75,135)
147 = f(22,75,140)
150 = f(24,102,132)
150 = f(71,73,138)
150 = f(75,100,125)
150 = f(76,86,132)
151 = f(46,47,148)
152 = f(24,80,144)
152 = f(62,66,144)
153 = f(17,102,136)
156 = f(65,87,142)
156 = f(78,104,130)
159 = f(3,121,131)
159 = f(87,102,132)
160 = f(56,112,136)
160 = f(69,123,124)
162 = f(18,108,144)
162 = f(36,57,159)
162 = f(50,96,148)
162 = f(59,93,148)
162 = f(81,108,135)
164 = f(8,68,160)
164 = f(24,128,132)
164 = f(38,120,138)
164 = f(69,99,146)
167 = f(12,86,159)
168 = f(56,106,150)
168 = f(84,112,140)
168 = f(108,114,126)
170 = f(96,107,141)
170 = f(100,122,128)
171 = f(19,114,152)
171 = f(27,90,162)
171 = f(54,80,163)
171 = f(107,108,136)
172 = f(1,135,138)
174 = f(40,108,158)
174 = f(45,126,147)
174 = f(47,97,162)
174 = f(52,110,156)
174 = f(66,90,162)
174 = f(76,96,158)
174 = f(87,116,145)
175 = f(28,119,154)
176 = f(25,92,167)
176 = f(42,86,168)
176 = f(50,62,172)
176 = f(64,92,164)
177 = f(48,137,142)
178 = f(34,80,172)
178 = f(48,133,147)
179 = f(17,57,177)
+++
180 = f(20,120,160)
180 = f(50,76,174)
180 = f(63,126,153)
180 = f(90,120,150)
180 = f(116,118,138)
+++
180 = f(20,120,160)
180 = f(50,76,174)
180 = f(63,126,153)
180 = f(90,120,150)
180 = f(116,118,138)
181 = f(108,109,150)
184 = f(12,144,148)
184 = f(108,120,148)
185 = f(57,68,180)
185 = f(68,113,166)
186 = f(18,121,167)
186 = f(64,108,170)
186 = f(93,124,155)
187 = f(58,131,160)
188 = f(115,122,149)
189 = f(21,46,188)
189 = f(21,126,168)
190 = f(30,100,180)
190 = f(56,133,163)
191 = f(39,146,156)
192 = f(34,123,173)
192 = f(38,106,180)
192 = f(96,128,160)
193 = f(25,68,190)
194 = f(90,138,158)
196 = f(126,133,147)
197 = f(53,58,194)
198 = f(22,132,176)
198 = f(27,46,197)
198 = f(73,135,170)
198 = f(99,132,165)
199 = f(6,127,180)
200 = f(32,136,176)
200 = f(70,140,170)
+++
